tests/test_core.py

```diff
@@ -8270,6 +8270,19 @@ def test_gl_main_module(self):
     self.set_setting('MAIN_MODULE')
     self.do_runf(path_from_root('tests', 'core', 'test_gl_get_proc_address.c'))
 
+  def test_dyncalls(self):
```

Comment:
```suggestion
  @also_with_wasm_bigint
  def test_dyncalls(self):
```
That will verify the emscripten.py changes here.